.travis.yml

```diff
@@ -21,4 +21,4 @@ cache:
     - ~/.cache
 
 script:
-  - ./run_tests.sh $GOVERSION
+  - ./run_tests.sh docker
```

Comment:
Perhaps it's my laptop, but I can only get ./run_tests.sh to work with `./run_tests.sh 1.11`, not `./run_tests.sh docker` or `./run_tests.sh podman`.